### PR TITLE
Update all web Dependencies 

### PR DESCRIPTION
**Pull request by Fish2**
Not Sure if this is worth it or will help but I updated all web Dependencies. 

Note: Updated them to latest and in some cases to the newest without big changes/breaking sab.

Bootstrap Typeahead 3.1.0 To 4.0.2
JQuery Form Plugin 3.51.0 To 4.2.2
JQuery Qrcode 0.12.0 To 0.14.0
Moment 2.18.1 To  2.19.2
Six 1.10.0 To 1.11.0

### CODE REVIEW

**Comment by Safihre** [comment]
While some modules can be updated without problem, for the big ones there's a reason they are older:
For example we use a quite heavily modified version of cherrypy. And updating jquery results in slower initial page loads in Glitter (queue will first flash empty before being filled, this is not the case on older versions). 
Same for bootstrap, the newer version the icons (glyphicons) are less sharp and therefore we use the old one.

**Comment by Fish2** [comment]
Ok let me know i can always remove some of the updates

File: SABnzbd.py [hunk 1]
```diff
@@ -54,9 +54,6 @@
     sys.exit(1)
 
 import cherrypy
-if [int(n) for n in cherrypy.__version__.split('.')] < [8, 1, 2]:
```
Comment (thezoggy) [inline SABnzbd.py]: removing this check is not ideal. while we provide cherrypy with the source.. not everyone uses it. this test is to safeguard that outside versions have the features we require and work with the code we have. our version is just modified to reduce bugs.

**Comment by Safihre** [comment]
I think we could use these:
Bootstrap Typeahead 3.1.0 To 4.0.2
JQuery Form Plugin 3.51.0 To 4.2.2
JQuery Qrcode 0.12.0 To 0.14.0
Moment 2.18.1 To 2.19.2
Six 1.10.0 To 1.11.0

**Comment by Fish2** [comment]
@Safihre ok thanks would you like me to remove the other comments or create a new pull request with them updates ?

**Comment by Safihre** [comment]
Rebase with those commits removed is good 👍

**Comment by Fish2** [comment]
All Done Thanks

**Comment by Safihre** [comment]
Looking good.
Now could you maybe only include `moment.js` locale's that we actually use and have them have the filenames identical to our language codes? 
To save some space and without right filename they don't get loaded :)
```
da
de
en
fi
fr
he
nb
nl
pl
pt-br
ro
ru
sr
sv
zh-cn
``

**Comment by Fish2** [comment]
@thezoggy Yer the filebrowser works its all good

**Comment by Fish2** [comment]
@Safihre Ok nice thats all cleaned up

**Comment by Safihre** [comment]
If you could add `he.js` (new language) then it's perfect, I tested all the other stuff 👍

**Comment by Safihre** [comment]
👍

**Comment by Fish2** [comment]
@Safihre Nice Cheers. Yer I have also tested 👍

**Comment by thezoggy** [comment]
should rebase the moment changes to keep history clean.

**Pull request merged**